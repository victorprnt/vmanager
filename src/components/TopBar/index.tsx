import * as S from './styles'

const TopBar = () => (
  <S.Wrapper>
    <h1>TopBar</h1>
  </S.Wrapper>
)

export default TopBar
