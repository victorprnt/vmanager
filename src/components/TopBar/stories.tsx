// import { Story, Meta } from '@storybook/react'
import TopBar from '.'

// export default {
//   title: 'TopBar',
//   component: TopBar
// } as Meta

// export const Default: Story = () => <TopBar />
