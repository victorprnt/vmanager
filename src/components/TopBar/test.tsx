import { render, screen } from '@testing-library/react'

import TopBar from '.'

describe('<TopBar />', () => {
  it('should render the heading', () => {
    const { container } = render(<TopBar />)

    expect(screen.getByRole('heading', { name: /TopBar/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
