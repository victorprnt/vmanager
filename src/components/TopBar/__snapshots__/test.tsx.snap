// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopBar /> should render the heading 1`] = `
<main
  class="sc-bczRLJ"
>
  <h1>
    TopBar
  </h1>
</main>
`;
